cloud: add a doc for the budget feature (#18315)
pingcap/docs: tidb-cloud/tidb-cloud-budget.md

When your monthly actual costs exceed the percentage thresholds of your specified budget, alert emails are sent to your organization owners and billing administrators. These notifications help you stay informed and take proactive measures to manage your spending, aligning your expenses with your budget.

TiDB Cloud provides two types of budgets to help you track your spending:

- **Serverless Spending Limit** budget: for each TiDB Serverless scalable cluster, TiDB Cloud automatically creates a **Serverless Spending Limit** budget. This budget helps you track the actual cost against the [spending limit](/tidb-cloud/manage-serverless-spend-limit.md) configured on that cluster. It includes three threshold rules: 75%, 90%, and 100% of the budget, which are not editable.

- **Custom** budget: you can create custom budgets to track actual costs for an entire organization or specific projects. For each budget, you can specify a budget scope, set a target spending amount, and configure alert thresholds. After creating a custom budget, you can compare your monthly actual costs with your planned costs to ensure you stay within budget.

## Prerequisites

> To view, create, and edit budgets of your organization, you must be in the `Organization Owner` or `Organization Billing Admin` role of your organization.

Author: ```suggestion
To view, create, edit, or delete budgets of your organization or projects, you must be in the `Organization Owner` or `Organization Billing Admin` role of your organization.
```

## View the budget information

To view the budget page of your organization, take the following steps:

1. In the lower-left corner of the TiDB Cloud console, click <MDSvgIcon name="icon-top-organization" />, and then click **Billing**.

    > **Note:**
    >
    > If you are in multiple organizations, switch to your target organization by clicking its name.

2. On the **Billing** page, click the **Budgets** tab.

For each budget, you can view its name, type, status, amount used, budget amount, period, and scope.

## Create a custom budget

To create a custom budget to monitor the spending of your organization or specific projects, take the following steps:

1. In the lower-left corner of the TiDB Cloud console, click <MDSvgIcon name="icon-top-organization" />, and then click **Billing**.

    > **Note:**
    >
    > If you are in multiple organizations, switch to your target organization by clicking its name.

2. On the **Billing** page, click the **Budgets** tab.

> 3. On the **Budgets** page, click **Create Custom Budget**. You can create up to 5 custom budgets.

Reviewer: ```suggestion
3. On the **Budgets** page, click **Create Custom Budget**. You can create up to five  custom budgets.
```

    - By default, TiDB Cloud provides three alert thresholds: 75%, 90%, and 100% of the budget amount. You can modify these percentages as needed.
    - To add a new alert threshold, click **Add alert threshold.**
    - To remove a threshold, click the delete icon next to the threshold.

7. Click **Create**.

## Edit a custom budget

> **Note:**
>
> The **Serverless Spending Limit** budget cannot be edited because it is automatically created by TiDB Cloud to help you track the cost of a TiDB Serverless scalable cluster against its [spending limit](/tidb-cloud/manage-serverless-spend-limit.md).

To edit a custom budget, take the following steps:

1. In the lower-left corner of the TiDB Cloud console, click <MDSvgIcon name="icon-top-organization" />, and then click **Billing**.

    > **Note:**
    >
    > If you are in multiple organizations, switch to your target organization by clicking its name.

2. On the **Billing** page, click the **Budgets** tab.

3. On the **Budgets** page, locate the row of your budget, click **...** in that row, and then click **Edit**.

> 

Reviewer: ```suggestion

4. Edit the custom budget information as needed. 
```

> ## Remove a custom budget

Author: ```suggestion
## Delete a custom budget
```

> **Note:**
>

> > The **Serverless Spending Limit** budget cannot be removed because it is automatically created by TiDB Cloud to help you track the cost of a TiDB Serverless scalable cluster against its [spending limit](/tidb-cloud/manage-serverless-spend-limit.md).

Author: ```suggestion
> The **Serverless Spending Limit** budget cannot be deleted because it is automatically created by TiDB Cloud to help you track the cost of a TiDB Serverless scalable cluster against its [spending limit](/tidb-cloud/manage-serverless-spend-limit.md).
```

> To delete a custom budget, take the following steps:

Reviewer: ```suggestion
To remove a custom budget, take the following steps:
```

1. In the lower-left corner of the TiDB Cloud console, click <MDSvgIcon name="icon-top-organization" />, and then click **Billing**.

    > **Note:**
    >
    > If you are in multiple organizations, switch to your target organization by clicking its name.

2. On the **Billing** page, click the **Budgets** tab.

> 3. On the **Budgets** page, locate the row of your budget, click **...** in that row, and then click **Remove**.

Author: ```suggestion
3. On the **Budgets** page, locate the row of your budget, click **...** in that row, and then click **Delete**.
```

2. On the **Billing** page, click the **Budgets** tab.

For each budget, you can view its name, type, status, amount used, budget amount, period, and scope.

## Create a custom budget

To create a custom budget to monitor the spending of your organization or specific projects, take the following steps:

1. In the lower-left corner of the TiDB Cloud console, click <MDSvgIcon name="icon-top-organization" />, and then click **Billing**.

    > **Note:**
    >
    > If you are in multiple organizations, switch to your target organization by clicking its name.

2. On the **Billing** page, click the **Budgets** tab.

3. On the **Budgets** page, click **Create Custom Budget**. You can create up to five  custom budgets.

4. Provide the budget basic settings.

    - **Name**: enter a name for the budget.
    - **Period**: select a time range for tracking costs. Currently, you can only select **Monthly**, which starts on the first day of each month and resets at the beginning of each month. TiDB Cloud tracks your actual spending during the time range against your budget amount (your planned spending).

>     - **Scope**: apply the scope to all projects (which means the entire TiDB Cloud organization) or a specific project as needed.

Reviewer: ```suggestion
    - **Budget scope**: apply the scope to all projects (which means the entire TiDB Cloud organization) or a specific project as needed.
```

5. Set the budget amount.

    - **Budget Amount**: enter a planned spending amount for the selected period.

>     - **Apply credits and discounts**: choose whether to apply credits and discounts to the running total cost. Credits are used to reduce the cost of your TiDB Cloud usage, and discounts are reductions in the regular price of TiDB Cloud service. When this option is enabled, the budget tracks the running total cost minus credits and discounts.

Reviewer: ```suggestion
    - **Apply credits**: choose whether to apply credits to the running total cost. Credits are used to reduce the cost of your TiDB Cloud usage. When this option is enabled, the budget tracks the running total cost minus credits.
    - **Apply discounts**: choose whether to apply discounts to the running total cost. Discounts are reductions in the regular price of TiDB Cloud service. When this option is enabled, the budget tracks the running total cost minus discounts.
```

    - By default, TiDB Cloud provides three alert thresholds: 75%, 90%, and 100% of the budget amount. You can modify these percentages as needed.
    - To add a new alert threshold, click **Add alert threshold.**
    - To remove a threshold, click the delete icon next to the threshold.

7. Click **Create**.

## Edit a custom budget

> **Note:**
>
> The **Serverless Spending Limit** budget cannot be edited because it is automatically created by TiDB Cloud to help you track the cost of a TiDB Serverless scalable cluster against its [spending limit](/tidb-cloud/manage-serverless-spend-limit.md).

To edit a custom budget, take the following steps:

1. In the lower-left corner of the TiDB Cloud console, click <MDSvgIcon name="icon-top-organization" />, and then click **Billing**.

    > **Note:**
    >
    > If you are in multiple organizations, switch to your target organization by clicking its name.

2. On the **Billing** page, click the **Budgets** tab.

3. On the **Budgets** page, locate the row of your budget, click **...** in that row, and then click **Edit**.

4. Edit the custom budget information as needed. 

## Delete a custom budget

> **Note:**
>

> > The **Serverless Spending Limit** budget cannot be deleted because it is automatically created by TiDB Cloud to help you track the cost of a TiDB Serverless scalable cluster against its [spending limit](/tidb-cloud/manage-serverless-spend-limit.md).

Author: ```suggestion
> - Once a custom budget is deleted, you will no longer receive any alert emails related to it.
> - The **Serverless Spending Limit** budget cannot be deleted because it is automatically created by TiDB Cloud to help you track the cost of a TiDB Serverless scalable cluster against its [spending limit](/tidb-cloud/manage-serverless-spend-limit.md).
```

## Prerequisites

To view, create, edit, or delete budgets of your organization or projects, you must be in the `Organization Owner` or `Organization Billing Admin` role of your organization.

## View the budget information

To view the budget page of your organization, take the following steps:

1. In the lower-left corner of the TiDB Cloud console, click <MDSvgIcon name="icon-top-organization" />, and then click **Billing**.

    > **Note:**
    >
    > If you are in multiple organizations, switch to your target organization by clicking its name.

2. On the **Billing** page, click the **Budgets** tab.

For each budget, you can view its name, type, status, amount used, budget amount, period, and scope.

## Create a custom budget

To create a custom budget to monitor the spending of your organization or specific projects, take the following steps:

1. In the lower-left corner of the TiDB Cloud console, click <MDSvgIcon name="icon-top-organization" />, and then click **Billing**.

    > **Note:**
    >
    > If you are in multiple organizations, switch to your target organization by clicking its name.

2. On the **Billing** page, click the **Budgets** tab.

> 3. On the **Budgets** page, click **Create Custom Budget**. You can create up to five  custom budgets.

Author: ```suggestion
3. On the **Budgets** page, click **Create Custom Budget**. You can create up to five custom budgets.
```
